cirq/linalg/decompositions_test.py

```diff
@@ -310,22 +310,23 @@ def test_kak_repr():
         single_qubit_operations_after=(np.eye(2), cirq.unitary(cirq.Z)),
     ))
 
-    assert repr(cirq.KakDecomposition(
-        global_phase=1,
-        single_qubit_operations_before=(cirq.unitary(cirq.X),
-                                        cirq.unitary(cirq.Y)),
-        interaction_coefficients=(0.5, 0.25, 0),
-        single_qubit_operations_after=(np.eye(2), cirq.unitary(cirq.Z)),
-    )) == """
+    assert repr(
+        cirq.KakDecomposition(
+            global_phase=1,
+            single_qubit_operations_before=(cirq.unitary(cirq.X),
+                                            cirq.unitary(cirq.Y)),
+            interaction_coefficients=(0.5, 0.25, 0),
+            single_qubit_operations_after=(np.eye(2), cirq.unitary(cirq.Z)),
+        )) == """
 cirq.KakDecomposition(
     interaction_coefficients=(0.5, 0.25, 0),
     single_qubit_operations_before=(
-        np.array([[0j, (1+0j)], [(1+0j), 0j]]),
-        np.array([[0j, -1j], [1j, 0j]]),
+        np.array([[0j, (1+0j)], [(1+0j), 0j]], dtype=np.complex128),
+        np.array([[0j, -1j], [1j, 0j]], dtype=np.complex128),
     ),
     single_qubit_operations_after=(
-        np.array([[1.0, 0.0], [0.0, 1.0]]),
-        np.array([[(1+0j), 0j], [0j, (-1+0j)]]),
+        np.array([[1.0, 0.0], [0.0, 1.0]], dtype=np.float64),
+        np.array([[(1+0j), 0j], [0j, (-1+0j)]], dtype=np.complex128),
```

Comment:
What's up with these kak changes?  They seem unrelated.

Reply:
It's due to the proper_repr fix for numpy arrays, caught by adding tests that it wasn't including the dtype